MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,11 +2,197 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    
+    @IBOutlet weak private var counterLabel: UILabel!
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    
+
+
+    private var currentQuestionIndex: Int = 0
+    private lazy var currentQuestion  = questions[currentQuestionIndex]
+    private var correctAnswer: Int = 0
+    private var allowAnswer: Bool = true
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        show(quiz: convert(model: currentQuestion))
+        
+    }
+    
+    private func show(quiz step: QuizStepViewModel) {
+        counterLabel.text = step.questionNumber
+        imageView.image = step.image
+        textLabel.text = step.question
+        
+        
+    }
+    
+    private func convert(model: QuizQuestion) -> QuizStepViewModel {
+        return QuizStepViewModel(image: UIImage(named: model.image) ?? UIImage(),
+                                 question: model.text,
+                                 questionNumber: "\(currentQuestionIndex+1)/\(questions.count)")
+    }
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
+        imageView.layer.cornerRadius = 20
+                       
+        if isCorrect == true {
+            imageView.layer.borderColor = UIColor(named: "YP Green")?.cgColor
+            correctAnswer += 1
+        } else { imageView.layer.borderColor = UIColor(named: "YP Red")?.cgColor
+        }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {self.showNextQuestionOrResults()
+        }
```

Comment:
Содержимое замыкания лучше писать с новой строки, плюс здесь поехало форматриование : выдели класс и зажми control+i
```suggestion
        }
        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
            self.showNextQuestionOrResults()
        }
```